src/System.Private.ServiceModel/tools/scripts/TestRootCertificateInstaller.cmd

```diff
@@ -0,0 +1,35 @@
+@echo off
+setlocal
+
+REM 
+REM This script invokes the TestRootCertificateInstaller.exe, which fetches the CA certificate from the test service
+REM to the location specified on the command line
+REM 
+REM Note that this isn't needed on Windows to install the CA certs, as the test client's CertificateManager can 
+REM install this automatically; the tool is primarily used for Linux, but it's sometimes useful to be able to 
+REM obtain the CA cert from the server directly 
+REM 
+
+pushd  %~dp0..\..\..\..
+REM Locate any CoreRun.exe to be able to execute TestClientCertificateInstaller.exe
+REM Because TestClientCertificateInstaller itself is a test project, we are guaranteed
```

Comment:
When I was trying to get this to run on Ubuntu, I found CoreRun.exe lives under Tools.  It might simplify things just to use it from there rather than searching for it as we once had to do.